getting_started/build-guide/build-from-scratch.rst

```diff
@@ -157,7 +157,9 @@ Clone, build, and install alsa-utils.
    # To install alsa-utils locally
    ./gitcompile --prefix=$HOME/local \
                 --with-alsa-inc-prefix=$HOME/local/include \
-		--with-alsa-prefix=$HOME/local/lib
+		--with-alsa-prefix=$HOME/local/lib \
+		--with-systemdsystemunitdir=$HOME/local/lib/systemd \
+		--with-udev-rules-dir=$HOME/locallib/udev
```

Comment:
Argh... too hasty. Missing forward slash between local and lib on the last line.